.gitignore

```diff
@@ -24,3 +24,4 @@ vendor/
 juju-backup-*.tar.gz
 .editorconfig
 scripts/mgo-run-txn/mgo-run-txn
+tests/tmp.*
```

Comment:
If a test fails, then we leave these around for investigation, but can be added to juju repo by accident.